guides/common/modules/proc_adding-custom-deb-repository-example-for-debian-11.adoc

```diff
@@ -14,17 +14,17 @@ For more information, see xref:Creating_a_Custom_Product_{context}[].
 . On the *Repositories* tab, click *New Repository* to create three repositories of type `deb` as follows:
 +
 * *Debian 11 main*
-** *URL*: `\http://ftp.debian.org/debian/`
+** *Upstream URL*: `\http://ftp.debian.org/debian/`
```

Comment:
Unrelated to this PR, but when I open https://ftp.debian.org/ it looks like it's really `deb.debian.org`. Perhaps it should all be consistent (now only security mentions `deb.debian.org`).

Reply:
Good idea. Changed all instances.